### ISSUE TITLE
Weird plugins ecosystem.

### ISSUE THREAD

**Issue by jensens** [plone/plone.app.theming#46]
With ``plone.app.theming.plugins.*`` theres a way to plugin own transforms. 

First using the sub-namenspace is possible, but nowhere else in plone we do so. Declaring them this way would enabled packages like `plone.app.theming.plugins.somename``.  I'd like to get rid of it.  I also doubt anybody uses them.

Second, if we need plugins we can use ZCA or entry-points in order to declare them, here a third way to do the same is introduced somehow.

**Comment by vangheem** [comment]
I agree. https://pypi.python.org/pypi?%3Aaction=search&term=plone.app.theming.plugins&submit=search brings up nothing.

Also, Plone 5 is a reasonable cut off point for that behavior IMO.

Please remove.

**Comment by gforcada** [comment]
More than two ways of doing something? /me loves it (sarcasm).

And indeed, as @vangheem points out, Plone5 is the best time to get rid of these things. Thanks @jensens for taking a look at it!

**Comment by jensens** [comment]
Do we need this plugins at all/ is this a classical case of over engineering?

**Comment by jensens** [comment]
btw, there are no tests for the plugin system. also if i remove the directory and zcml include plus some relevant lines in utils.py nothing seems to break.

**Comment by datakurre** [comment]
I don't know about ``plone.app.theming.plugins.*``, but IMHO the plugins system described in ``plone.app.theming.interfaces.IThemePlugin`` should be preserved.
